precompiles/werc20/werc20.go

```diff
@@ -137,14 +135,11 @@ func (p Precompile) Run(evm *vm.EVM, contract *vm.Contract, readOnly bool) (bz [
 }
 
 // IsTransaction checks if the given methodID corresponds to a transaction or query.
-func (p Precompile) IsTransaction(methodID string) bool {
-	switch methodID {
-	case cmn.FallbackMethod,
-		cmn.ReceiveMethod,
```

Comment:
But please aware `fallback` and `receive` still execute as tx because of the other logic

Reply:
Indeed they will but that case is handled separately when calling without a `data` but with a `value` field.

Reply:
Sure
> The fallback function is executed on a call to the contract if none of the other functions match the given function signature, or if no data was supplied at all and there is no [receive Ether function](https://docs.soliditylang.org/en/latest/contracts.html#receive-ether-function)